packages/edit-site/src/components/use-edited-entity-record/index.js

```diff
@@ -26,7 +26,8 @@ export default function useEditedEntityRecord( postType, postId ) {
 				usedPostType,
 				usedPostId
 			);
-			const _isLoaded = !! usedPostId;
+			const _isLoaded =
+				!! usedPostId && _record && Object.keys( _record ).length > 0;
```

Comment:
Do you know if there's a reason we're not using `hasFinishedResolution` for the `isLoaded` status here?